app/(white_navbar)/explore/RetroPGF5/_component/BreadcrumbExplore.tsx

```diff
@@ -0,0 +1,22 @@
+import ChevronRight from '@carbon/icons-react/lib/ChevronRight'
+import Link from 'next/link'
+
+export default function BreadcrumbExplore({
+  secondText,
+}: {
+  secondText: string
+}) {
+  return (
+    <div className="flex flex-wrap gap-2 p-2 md:p-0">
+      <Link
+        href={'/explore'}
+        className="text-xs font-normal text-gray-800 hover:text-red-500 hover:underline"
+      >
+        {' '}
+        Explore{' '}
```

Comment:
Why don't it just be this?
```
<Link
        href={'/explore'}
        className="text-xs font-normal text-gray-800 hover:text-red-500 hover:underline"
      >
        Explore
      </Link>
```

Reply:
Because we need whitespace

Reply:
I don't think it make any extra whitespace
![image](https://github.com/user-attachments/assets/ff4a08f2-711e-48c5-84fa-54f8be64a746)
![image](https://github.com/user-attachments/assets/c9ec4958-1aa0-422b-bb87-ffe4a9503ef5)

Reply:
Ok, then I will remove it